composer.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "baopham/dynamodb",
-    "description": "Eloquent syntax for DynamoDB",
+    "name": "thebatclaudio/dynamodb",
```

Comment:
are these changes needed here?

Reply:
I changed this because I needed these changes, so I forked the library and republished on composer. I will remove this.